homeassistant/components/fritz/switch.py

```diff
@@ -218,11 +218,18 @@ def port_entities_list(
 
         # We can only handle port forwards of the given device
         if portmap["NewInternalClient"] == local_ip:
+            port_desc = portmap["NewPortMappingDescription"]
```

Comment:
maybe use `port_name` to be consistent with the class?

Reply:
Done